From 86ae52d4a9988958aec3ec9955f3e2190bfa81c7 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 15 Feb 2017 18:13:05 -0800
Subject: [PATCH] MANATEE-288 Manatee did not seem to reconnect to ZK after
 loss Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved by: David
 Pacheco <dap@joyent.com>

---
 bin/zkConnTest.js         | 2 +-
 lib/adm.js                | 2 +-
 lib/zookeeperMgr.js       | 2 +-
 package.json              | 2 +-
 test/integ.test.js        | 2 +-
 test/zookeeperMgr.test.js | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/bin/zkConnTest.js b/bin/zkConnTest.js
index 8eb74c3..c16ffb1 100755
--- a/bin/zkConnTest.js
+++ b/bin/zkConnTest.js
@@ -11,7 +11,7 @@
  */
 var once = require('once');
 var vasync = require('vasync');
-var zkClient = require('node-zookeeper-client');
+var zkClient = require('joyent-zookeeper-client');
 
 var connStr = process.argv[2];
 var timeout = process.argv[3] ? parseInt(process.argv[3], 10) : 0;
diff --git a/lib/adm.js b/lib/adm.js
index 632697e..303322e 100644
--- a/lib/adm.js
+++ b/lib/adm.js
@@ -35,7 +35,7 @@ var util = require('util');
 var vasync = require('vasync');
 var VError = require('verror');
 var zfs = require('./zfsClient');
-var zk = require('node-zookeeper-client');
+var zk = require('joyent-zookeeper-client');
 
 var LOG = bunyan.createLogger({
     name: 'manatee-adm',
diff --git a/lib/zookeeperMgr.js b/lib/zookeeperMgr.js
index b264fff..7fcfba5 100644
--- a/lib/zookeeperMgr.js
+++ b/lib/zookeeperMgr.js
@@ -24,7 +24,7 @@ var EventEmitter = require('events').EventEmitter;
 var once = require('once');
 var util = require('util');
 var vasync = require('vasync');
-var zkClient = require('node-zookeeper-client');
+var zkClient = require('joyent-zookeeper-client');
 
 /**
  * An abstraction over zookeeper for the things that only manatee cares about.
diff --git a/package.json b/package.json
index 43dee53..6624337 100644
--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
         "manatee-state-machine": "git://github.com/joyent/manatee-state-machine#master",
         "manta": "1.2.6",
         "node-uuid": "1.4.1",
-        "node-zookeeper-client": "0.2.1",
+        "joyent-zookeeper-client": "0.2.2",
         "once": "1.3.0",
         "pg": "3.0.3",
         "posix": "1.0.3",
diff --git a/test/integ.test.js b/test/integ.test.js
index 6f66363..d912a68 100644
--- a/test/integ.test.js
+++ b/test/integ.test.js
@@ -35,7 +35,7 @@ var util = require('util');
 var uuid = require('node-uuid');
 var vasync = require('vasync');
 var verror = require('verror');
-var zk = require('node-zookeeper-client');
+var zk = require('joyent-zookeeper-client');
 
 var FS_PATH_PREFIX = process.env.FS_PATH_PREFIX || '/var/tmp/manatee_tests';
 var ZK_URL = process.env.ZK_CONN_STR || 'localhost:2181';
diff --git a/test/zookeeperMgr.test.js b/test/zookeeperMgr.test.js
index 161bcfa..065eba4 100644
--- a/test/zookeeperMgr.test.js
+++ b/test/zookeeperMgr.test.js
@@ -24,7 +24,7 @@ var bunyan = require('bunyan');
 var childProcess = require('child_process');
 var fs = require('fs');
 var vasync = require('vasync');
-var zkClient = require('node-zookeeper-client');
+var zkClient = require('joyent-zookeeper-client');
 var ZkMgr = require('../lib/zookeeperMgr');
 
 /**
-- 
2.21.0

